utilization.md

```diff
@@ -157,14 +155,8 @@ Kokkos::View<double*, Kokkos::HostSpace> hostView("hostView", numberOfElements);
 // Shared space
 Kokkos::View<double*, Kokkos::SharedSpace> sharedView("sharedView", numberOfElements);
 
-// Scratch memory space
-Kokkos::parallel_for(
-    Kokkos::TeamPolicy<>(leagueSize, teamSize),
-    KOKKOS_LAMBDA (const Kokkos::TeamPolicy<>::member_type& team) {
-        // Allocate scratch memory for each team
-        Kokkos::View<double*, Kokkos::ScratchMemorySpace> scratchView(team.team_scratch(1), scratchSize);
-    }
-);
+// Shared host pinned space
+Kokkos::View<double*, Kokkos::SharedHostPinnedSpace> sharedHostPinnedView("sharedHostPinnedView", numberOfElements);
```

Comment:
Why did you remove the scratch space example ?

Reply:
This is more from Damien's feedback in June than specific to 4.5. From his words, the scratch space is not a "real" memory space, given that it can only be used in very specific situations.

Reply:
Fine for me